### PR TITLE
Addressing Ben Campbell's IESG Review Comments

### PR DESCRIPTION
**Pull request by seanturner**
Changes based on [Ben's comments](https://mailarchive.ietf.org/arch/msg/rtcweb/mDXe6mxzyWXAi-_b_u7-0o86mJQ).

### CODE REVIEW

**Comment by seanturner** [comment]
@ekr @adamroach @martinthomson @nils-ohlmeier PTAL

Also, there were two I wasn't entirely sure how to address:

    §5.1.4:
    - "In this
    case, the established identity SHOULD be applied to existing DTLS
    connections as well as new connections established using one of those
    fingerprints."

    Applied by the recipient? (consider active voice). Also, why not MUST? Don't
    unexpected things happen if the recipient doesn't do this?

    §6.2:
    - "Because HTTP origins cannot be securely established against network
    attackers, implementations MUST NOT allow the setting of permanent
    access permissions for HTTP origins. Implementations MUST refuse all
    permissions grants for HTTP origins."

    (nit-ish) - The MUST NOT seems non-constraining considering the last sentence.
    Or am I reading that sentence wrong?

**Comment by martinthomson** [comment]
> In this
case, the established identity can be attributed to existing DTLS
connections with fingerprints that match any of the included fingerprints, plus any new connections established using one of those
fingerprints.

This doesn't need a SHOULD, it's axiomatic.

The 6.2 thing is a interesting.  The second statement appears to be merely a restatement of the first, for emphasis, but it is a stronger statement.  So Ben is right, the first sentence is unnecessary.

> Because HTTP origins cannot be securely established against network
attackers, implementations MUST refuse all requests for
permission grants from HTTP origins.

File: draft-ietf-rtcweb-security-arch.xml [hunk 1]
```diff
@@ -31,6 +30,7 @@
 <!ENTITY RFC8261 SYSTEM "http://xml2rfc.ietf.org/public/rfc/bibxml/reference.RFC.8261.xml">
 <!ENTITY RFC8445 SYSTEM "http://xml2rfc.ietf.org/public/rfc/bibxml/reference.RFC.8445.xml">
 
+<!ENTITY I-D.ietf-rtcweb-jsep SYSTEM "http://xml2rfc.ietf.org/public/rfc/bibxml3/reference.I-D.ietf-rtcweb-overview-19.xml">
```
Comment (martinthomson) [inline draft-ietf-rtcweb-security-arch.xml]: This entity is wrong.  Did you mean I-D.ietf-rtcweb-overview ?  Did you mean to reference -19 specifically, as opposed to the latest?

File: draft-ietf-rtcweb-security-arch.xml [hunk 2]
```diff
@@ -94,9 +94,10 @@
 
     <section title="Introduction" anchor="sec.introduction">
       <t>
-        The Real-Time Communications on the Web (WebRTC) working group is tasked
-        with standardizing protocols for real-time communications between Web
-        browsers. The major use cases for WebRTC technology are real-time audio
+        The Real-Time Communications on the Web (RTCWEB) working group
+        standardized protocols for real-time communications between Web
+        browsers, generally called "WebRTC" <xref traget="I-D.ietf-rtcweb-overview"/>.
```
Comment (martinthomson) [inline draft-ietf-rtcweb-security-arch.xml]:
```suggestion
        browsers, generally called "WebRTC" <xref target="I-D.ietf-rtcweb-overview"/>.
```

File: draft-ietf-rtcweb-security-arch.xml [hunk 3]
```diff
@@ -129,7 +130,7 @@
         in <xref target="fig.multidomain"/>.  The protocol to be used between
         the domains is not standardized by WebRTC, but given the installed base
         and the form of the WebRTC API is likely to be something SDP-based like
-        SIP.
+        SIP or something like Extensible Messaging and Presence Protocol (XMPP).
```
Comment (martinthomson) [inline draft-ietf-rtcweb-security-arch.xml]: \[citation needed]

File: draft-ietf-rtcweb-security-arch.xml (hunk 1 above)
Comment (seanturner) [reply draft-ietf-rtcweb-security-arch.xml]: I haven't got a clue why none of the recent PRs here will build :(

**Comment by seanturner** [comment]
@martinthomson okay fixed these points.  On to Ben K's.

**Comment by ekr** [comment]
I don't think this "SHOULD" -> "can" for identities is right. Actually, I think it needs to be a MUST.

**Pull request merged**